books/free-programming-books-subjects.md

```diff
@@ -221,6 +222,10 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Theory and Applications for Advanced Text Mining](http://www.intechopen.com/books/theory-and-applications-for-advanced-text-mining)
 
 
+### DevOps
+* [Phoenix Project Excerpt](https://drive.google.com/file/d/1l8mk9vqaIesR5GuS7n8v9i3GkDyAHunj/view?usp=sharing) - Gene Kim, George Spafford, and Kevin Behr
```

Comment:
Reading [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) guidelines...

> - we don't accept files hosted on google drive, dropbox, mega, scribd, issuu and other similar file upload platforms
> - use the link with the most authoritative source (meaning the author's website is better than the editor's website, which is better than a third party website)

Maybe the link provided by the owners website? It's free, search for it